app/assets/stylesheets/_category.scss

```diff
@@ -0,0 +1,13 @@
+.category {
+  padding-left: 10px;
```

Comment:
Properties should be ordered list-style-type, margin-left, margin-right, padding-left, padding-right